runtime-sofa-boot-starter/src/main/java/com/alipay/sofa/runtime/spring/parser/AbstractContractDefinitionParser.java

```diff
@@ -40,15 +40,15 @@
 public abstract class AbstractContractDefinitionParser extends AbstractSingleBeanDefinitionParser
                                                                                                  implements
                                                                                                  SofaBootTagNameSupport {
-    private static final String INTERFACE_ELEMENT           = "interface";
-    private static final String INTERFACE_PROPERTY          = "interfaceType";
-    private static final String BEAN_ID_ELEMENT             = "id";
-    private static final String BEAN_ID_PROPERTY            = "beanId";
-    private static final String UNIQUE_ID_ELEMENT           = "unique-id";
-    private static final String UNIQUE_ID_PROPERTY          = "uniqueId";
-    private static final String ELEMENTS                    = "elements";
-    private static final String REPEAT_REFER_LIMIT_ELEMENT  = "repeatReferLimit";
-    private static final String REPEAT_REFER_LIMIT_PROPERTY = "repeatReferLimit";
+    protected static final String INTERFACE_ELEMENT           = "interface";
+    protected static final String INTERFACE_PROPERTY          = "interfaceType";
+    protected static final String BEAN_ID_ELEMENT             = "id";
+    protected static final String BEAN_ID_PROPERTY            = "beanId";
+    protected static final String UNIQUE_ID_ELEMENT           = "unique-id";
+    protected static final String UNIQUE_ID_PROPERTY          = "uniqueId";
+    protected static final String ELEMENTS                    = "elements";
+    protected static final String REPEAT_REFER_LIMIT_ELEMENT  = "repeatReferLimit";
+    protected static final String REPEAT_REFER_LIMIT_PROPERTY = "repeatReferLimit";
 
```

Comment:
Since the logic moves to AbstractContractDefinitionParser,  I think we do not need to change `private` to `protected`